Source/WebCore/page/Quirks.cpp

```diff
@@ -1111,6 +1130,10 @@ static bool isStorageAccessQuirkDomainAndElement(const URL& url, const Element&
         || element.classNames().contains("sb-signin-button"_s));
     }
 
+    // Gizmodo login case: rdar://106782128.
+    if (url.host() == "gizmodo.com"_s)
```

Comment:
Are we sure the host will always be gizmodo.com? Or should we look at the registrable domain to also cover subdomains?

Reply:
> We have expressed in multiple standards conversations that the limit of domains under which a same-party SSO domain gets cookie access should be 5.

Do you suggest a change here? This is not part of the patch and it's added in https://trac.webkit.org/changeset/263383/webkit for the old quirk, so I don't know much about it.

> 
Are we sure the host will always be gizmodo.com? Or should we look at the registrable domain to also cover subdomains?
I tested only on gizmodo.com for this radar, and it's "gizmodo.com".

Reply:
> > We have expressed in multiple standards conversations that the limit of domains under which a same-party SSO domain gets cookie access should be 5.
> 
> Do you suggest a change here? This is not part of the patch and it's added in https://trac.webkit.org/changeset/263383/webkit for the old quirk, so I don't know much about it.

No change requested. I just mentioned this since Karl brought up a large set of domains.

> > Are we sure the host will always be gizmodo.com? Or should we look at the registrable domain to also cover subdomains?
> 
> I tested only on gizmodo.com for this radar, and it's "gizmodo.com".

OK. We can change the quirk if we learn of any subdomain use.